src/settings/VoucherNumber/SettingsVoucherNumberForm.js

```diff
@@ -31,8 +31,8 @@ const SettingsVoucherNumberForm = ({ onReset, firstSequenceNumber }) => (
           component={TextField}
           label={<FormattedMessage id="ui-invoice.settings.voucherNumber.startNumber" />}
           name="sequenceNumber"
-          disabled
-          required
+          type="number"
+          disabled={!allowVoucherNumberEdit}
```

Comment:
I doubt that `allowVoucherNumberEdit` controls the settings. I suppose, it's for edit voucher form. Could you please specify why you've decided to implement it here?